src/main/kotlin/io/easybreezy/project/model/issue/Solution.kt

```diff
@@ -0,0 +1,88 @@
+package io.easybreezy.project.model.issue
+
+import io.easybreezy.infrastructure.event.project.issue.DueDateChanged
+import io.easybreezy.infrastructure.event.project.issue.Reassigned
+import io.easybreezy.infrastructure.event.project.issue.StatusUpdated
+import io.easybreezy.infrastructure.event.project.issue.WatchersUpdated
+import io.easybreezy.infrastructure.exposed.dao.AggregateRoot
+import io.easybreezy.infrastructure.exposed.dao.PrivateEntityClass
+import io.easybreezy.infrastructure.exposed.type.jsonb
+import io.easybreezy.infrastructure.serialization.UUIDSerializer
+import kotlinx.serialization.builtins.list
+import org.jetbrains.exposed.dao.EntityClass
+import org.jetbrains.exposed.dao.id.EntityID
+import org.jetbrains.exposed.dao.id.IdTable
+import org.jetbrains.exposed.sql.Column
+import org.jetbrains.exposed.sql.ResultRow
+import org.jetbrains.exposed.sql.`java-time`.datetime
+import java.time.LocalDateTime
+import java.util.UUID
+
+class Solution private constructor(id: EntityID<UUID>) : AggregateRoot<UUID>(id) {
```

Comment:
We've discussed decomposition. Continue on Monday with choices of type of decompositions.
My proposal Behavior(status) , Estimates, Participants(assign, owner)